homework-submission/homework-week2/src/App.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import './App.css';
+import arr from './todo.json';
+import DynamicList from './components/DynamicList';
+import TodoForm from './components/TodoForm';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { todo: arr };
+    this.deleteTodo = this.deleteTodo.bind(this);
+    this.updateTodoList = this.updateTodoList.bind(this);
+  }
+  deleteTodo(index) {
+    this.setState({ todo: this.state.todo.filter(elem => elem !== this.state.todo[index]) });
+  }
```

Comment:
better to make it like:
const todo = [...this.state.todo];
    todo.splice(index, 1);
    this.setState({      todo    });